linux/library/src/flutter_embedder.cc

```diff
@@ -98,19 +124,18 @@ static void GLFWOnFlutterPlatformMessage(const FlutterPlatformMessage *message,
   Json::Value response = state->plugin_handler->HandlePlatformMessage(
       channel, json, [window] { GLFWClearEventCallbacks(window); },
       [window] { GLFWAssignEventCallbacks(window); });
-  if (response.isNull()) {
-    return;
-  }
 
-  Json::StreamWriterBuilder writer_builder;
-  std::string output = Json::writeString(writer_builder, response);
-  FlutterPlatformMessage platform_message_response = {
-      .struct_size = sizeof(FlutterPlatformMessage),
-      .channel = channel.c_str(),
-      .message = reinterpret_cast<const uint8_t *>(output.c_str()),
-      .message_size = output.size(),
-  };
-  FlutterEngineSendPlatformMessage(state->engine, &platform_message_response);
+  if (!response.isNull()) {
+    Json::StreamWriterBuilder writer_builder;
+    std::string output = Json::writeString(writer_builder, response);
+    FlutterPlatformMessage platform_message_response = {
+        .struct_size = sizeof(FlutterPlatformMessage),
+        .channel = channel.c_str(),
+        .message = reinterpret_cast<const uint8_t *>(output.c_str()),
+        .message_size = output.size(),
+    };
+    FlutterEngineSendPlatformMessage(state->engine, &platform_message_response);
```

Comment:
Since this is all duplicate code with line 89, extract to a static helper? (Eventually I think this will move into Plugin, but I can revisit that when reworking the plugin API across platforms.)

Reply:
I think just calling the static function would be sufficient, no?  The code is the same, for sure.